### DISCUSSION TITLE
Welcome to Biip Discussions!

### DISCUSSION THREAD

**Discussion by jodal** [jodal/biip#66] [General]
## Welcome!

We’re using Discussions as a place to connect with other users of Biip. We hope that you:

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

To get started, comment below with an introduction of yourself and tell us about what you do with Biip.

**Comment by GustavoCosta1989** [comment]
Hi,
in _application_identifiers.json AI ​​7030 the pattern is incorrect
"^7030(\\d{3})(!%-?A-Z_a-z\\x22]{1,27})$"
correct:
"^7030(\\d{3})([!%-?A-Z_a-z\\x22]{1,27})$"

great work

**Reply by jodal** [reply]
I've created an issue to track this here: https://github.com/jodal/biip/issues/345
